src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/domain/Blog.java

```diff
@@ -63,4 +66,7 @@ public Boolean getDefaultApprovedValue() {
         }
     }
 
+    public String getStatus() {
+        return active ? "Active" : "Deactivated";
```

Comment:
this is only for display in view, maybe I should rename it to be more clear? any suggestion?